### ISSUE TITLE
Unbound Name: LC_*

### ISSUE THREAD

**Issue by bodinsamuel** [facebook/hhvm#4800]
Hi,

I already asked this question in the doc repo but it seems I could came from the type checker.
Using custom php constants for setlocale on a namespace is reported as error when using strict mode, while it correctly pass when used outside of a ns.

```php
<?hh // strict
namespace Myns;
class Hello
{
   public function locale()
   {
       setlocale(LC_ALL, "en_US");
   }
}
```
**Unbound name: LC_ALL (Naming[2049])**

All LC_* constants are triggering this error.

Thanks :)

**Comment by SiebelsTim** [comment]
cc @jwatzman for all the namespace issues.

**Comment by danslo** [comment]
Which version of HHVM? This doesn't happen to me on the latest nightly (which as of writing this is ``heads/master-0-g1ea69d060994735d7fec9e364d083f72fc900144``).

**Comment by bodinsamuel** [comment]
I'm using the last prebuilt package of hhvm for Debian 8

*hhvm --version*
HipHop VM 3.5.0 (rel)
Compiler: tags/HHVM-3.5.0-0-ga90f4733cfa0d8fefdafc7433f758f78cdc75424
Repo schema: b88b6fee8ef78ab638506d212abcbefe74a276ac

**Comment by danslo** [comment]
That's stable, so I guess you'll see the fix in 3.6.

**Comment by jwatzman** [comment]
Fixed by 338d1e5e5ee9b3a9889ccf2bb153f34623898cd5 which will be in 3.6.
